dkist/io/file_manager.py

```diff
@@ -419,6 +417,9 @@ def download(self, path=None, destination_endpoint=None, progress=True, wait=Tru
         label : `str`
             Label for the Globus transfer. If `None` then a default will be used.
         """
+        from dkist.net import conf as net_conf
```

Comment:
```suggestion
        # Import here to prevent triggering an import of `.net` with `dkist.dataset`.
        from dkist.net import conf as net_conf
```